HolyshiftService/DB models/HolyShiftUser.cs

```diff
@@ -0,0 +1,17 @@
+public class HolyShiftUser
+{
+    public string Id { get; set; }
+    public string UserName { get; set; }
```

Comment:
We need a unique index on this column.

Reply:
but I am not using the username as an identifier I use the email , so in the register if there Is a user with the same email the registration fails ,I mean in that case  there can be users with the same name right?

Reply:
I understand that we using email as the primary login identifier.
However, this could change in the future. We could decide to use Username as a secondary login identifier.

Adding a unique constraint then is going to be way harder.

However, if we have a unique constraint now, and then we decide that we will never need this uniqueness, it will be much simpler to remove the constraint without impacting our customers.